notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1173 @@
+{
```

Comment:
<p>This is close - a couple more comments:</p><p> </p><p>Please pull out the Java version into a separate var so that we only need to update it in one place going forward.</p><p> </p><pre class="ql-syntax" spellcheck="false">JAVA_VERSION = "8u332-b09"
FILE_NAME = f"openlogic-openjdk-{JAVA_VERSION}-linux-x64"
DOWNLOAD_LINK = (f"https://builds.openlogic.com/downloadJDK
                 /openlogic-openjdk/{JAVA_VERSION}/{TAR_FILE}")
</pre><p> </p><p>Please add a comment at the top of this cell that mentions why this step is necessary (test env doesn't have Java + PySpark but Dataproc does etc.)</p><p> </p><p>Please add a comment above the pip install that says something like "pin this version to match that of the Dataproc cluster used".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>